lib/checkWebHook.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var bufferEq = require('buffer-equal-constant-time');
+var Cesium = require('cesium');
+var crypto = require('crypto');
+
+var Check = Cesium.Check;
+var defined = Cesium.defined;
+
+var Settings = require('./Settings');
+module.exports = checkWebHook;
+
+function checkWebHook(req, res, next) { // eslint-disable-line no-unused-vars
+    Check.typeOf.object('req', req);
+
+    if (req.method !== 'POST') {
```

Comment:
I'm not sure why this check is needed, if the method isn't a post, none of our handlers should be called (assuming the handler is properly set up to only listen to POST).